lisp/forge-commands.el

```diff
@@ -185,6 +185,17 @@ Prefer a topic over a branch and that over a commit."
      (forge--format-url repo 'commit-url-format
                         `((?r . ,(magit-commit-p rev)))))))
 
+;;;###autoload
+(defun forge-copy-url-at-point-as-kill ()
+  "Copy the 'URL of the thing at point."
+  (interactive)
+  (if-let ((url (forge-get-url (or (forge-post-at-point)
+                                   (forge-current-topic)))))
+      (progn
+        (kill-new url)
+        (message "Copied %S" url))
+    (user-error "Nothing at point with a URL")))
```

Comment:
I think it is "an URL".

Reply:
That's not how it's spoken, at least – 'U' (specifically the phoneme 'yoo') is a weird one in regards to 'a'/'an'.

Reply:
English is weird...

My native language is German and I *think* that "the phoneme 'yoo'" is what I would write as "iu"; i.e. *two* vowels at the very beginning. But vowels are extremely weird in English--somehow two vowels even got switched (i/e)... so we'll leave this as-is and not spend any more time on it :stuck_out_tongue_closed_eyes: